dlp/snippets/src/main/java/dlp/snippets/DeIdentifyWithExceptionList.java

```diff
@@ -69,12 +73,31 @@ public static void deIdentifyWithExceptionList(String projectId, String textToDe
       CustomInfoType customInfoType =
           CustomInfoType.newBuilder().setInfoType(developerEmail).setDictionary(wordList).build();
 
+      ExclusionRule exclusionRule =
+          ExclusionRule.newBuilder()
+              .setDictionary(wordList)
+              .setMatchingType(MatchingType.MATCHING_TYPE_FULL_MATCH)
+              .build();
+
+      InspectionRule inspectionRule =
+          InspectionRule.newBuilder()
+              .setExclusionRule(exclusionRule)
+              .build();
+
       // Specify the word list custom info type and build-in info type the inspection will look for.
       InfoType emailAddress = InfoType.newBuilder().setName("EMAIL_ADDRESS").build();
```

Comment:
Is there a enum that can be used to set parameter in setName() instead of "EMAIL_ADDRESS" as string ?

Reply:
For every occurrence in the repo, "EMAIL_ADDRESS" is used explicitly. Looks like there's no enum defined. Converting it into enum will be followed by changes in other files as well.